docs/deployment/hotfix.md

```diff
@@ -0,0 +1,41 @@
+# Making a hotfix release
+
+This list outlines the manual steps needed to make a hotfix release of the
+`benefits` app.
+
+If `main` contains in-progress work that is not ready for production, and a hotfix
+is needed in production, a separate process to test the changes before deploying to prod must be undertaken.
+This is called a fotfix release.
```

Comment:
This is good, but maybe we can add a clause or small sentence about how this is different from a rollback release?

E.g. because for a hotfix, we know exactly what the issue is and it is a small/simple code change. A rollback is more like, we don't know what's going on or there was a security issue or something.

Reply:
Good idea! I added a bit of explanation here 66ac896dd7d86706aec826e48adf7ae6dfd49384